Source/ACE.Server/Network/Handlers/DDDHandler.cs

```diff
@@ -18,5 +20,31 @@ public static void DDD_EndDDD(ClientMessage message, Session session)
         {
             // We don't need to reply to this message.
         }
+
+        [GameMessage(GameMessageOpcode.DDD_RequestDataMessage, SessionState.WorldConnected)]
+        public static void DDD_RequestDataMessage(ClientMessage message, Session session)
+        {
+            // True DAT patching would be triggered by this msg, but as we're not supporting that, respond instead with warning and push to external download
+
+            var msg = "Your DAT files are incomplete.\nACEmulator does not support dynamic DAT updating at this time.\nPlease visit https://emulator.ac/how-to-play to download the complete DAT files.";
```

Comment:
should this be a safer default message perhaps, with a server option to customize the message?